ksql-serde/src/test/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroDeserializerTest.java

```diff
@@ -105,4 +118,121 @@ public void shouldCreateCorrectRow() {
         (100.0));
   }
 
+  @Test
+  public void shouldDeserializeCorrectly() {
+    SchemaRegistryClient schemaRegistryClient = new MockSchemaRegistryClient();
+    List columns = Arrays.asList(1511897796092l, 1l, "item_1", 10.0, new Double[]{100.0},
+                                 Collections.singletonMap("key1", 100.0));
+
+    GenericRow genericRow = new GenericRow(columns);
```

Comment:
add `@SuppressWarnings("unchecked")` to the method. Need to try and remove the noise from the build

Reply:
Done